### PR TITLE
Postgresql installed

### PR DESCRIPTION
**Pull request by Martinfree**
Trello card:
https://trello.com/c/PaWU0drm

- Changed database from mongodb to postgresql
- Relosved 'username' conflicts in admin.py. They appeared when new user is creating, because 'username' field must be unique

### CODE REVIEW

**Comment by pavlo-mk** [comment]
@Martinfree please add header to PR

**Review by pavlo-mk — CHANGES_REQUESTED** [review]
Please add description and trello card if it is. Thanks.

File: api/authentication/models.py [hunk 1]
```diff
@@ -88,6 +88,7 @@ def create_teacher(self, email, first_name=None, last_name=None, password=None):
 class StdUser(AbstractUser):
     objects = StdUserManager()
 
+    username = None
```
Comment (klekhaav) [inline api/authentication/models.py]: Check username field definition.

**Comment by klekhaav** [comment]
Looks fine, add link to trello, and add description as Pasha shared in slack

**Review by pavlo-mk — APPROVED** [review]

**Pull request merged**